internal/sqladapter/testing/adapter.go.tpl

```diff
@@ -117,7 +117,7 @@ func TestInsertReturning(t *testing.T) {
 	assert.NotZero(t, itemMap["id"], "Must not be zero after inserting")
 
 	itemStruct := struct {
-		ID int `db:"id,omitempty"`
```

Comment:
Not sure how other do this, let me check if we have some examples on the stdlib.